src/Stratis.Bitcoin.Features.Wallet/WalletTransactionHandler.cs

```diff
@@ -335,7 +340,11 @@ protected void AddCoins(TransactionBuildContext context)
                 // then it's safe to stop adding UTXOs to the coin list.
                 // The primary goal is to reduce the time it takes to build a trx
                 // when the wallet is bloated with UTXOs.
-                if (sum > totalToSend)
+
+                // Get to our total, and then check that we're a little bit over to account for tx fees.
+                // If it gets over totalToSend but doesn't hit this break, that's fine too.
+                // The TransactionBuilder will have a go with what we give it, and throw NotEnoughFundsException accurately if it needs to.
+                if (sum > totalToSend + PretendMaxFee)
```

Comment:
Would this mean I need over (n+1) strat to send n strat?

Reply:
Nope! If you have n+1 strat you will exit this loop early. If you have n, you will exit the loop normally and still add the coins underneath this.